tests/foreman/cli/test_repository.py

```diff
@@ -3134,3 +3134,25 @@ def test_positive_symlinks_sync(self, repo):
         Repository.synchronize({'id': repo['id']})
         repo = Repository.info({'id': repo['id']})
         assert int(repo['content-counts']['files']) > 1
+
+
+@pytest.mark.tier2
```

Comment:
this isn’t marked as stubbed, use `@pytest.mark.stubbed`

Reply:
oops, fixed now, thank you